wandb/apis/internal.py

```diff
@@ -137,6 +137,18 @@ def disabled(self):
         except configparser.Error:
             return False
 
+    def sync_spell(self, url, env=None):
+        """Syncs this run with spell"""
+        try:
+            env = env or os.environ
+            wandb.config.update({"spell_url": env.get("SPELL_RUN_URL")})
```

Comment:
We should probably set this under the _wandb namespace so it doesnt pollute user config. For example: `wandb.config.set_wandb()`